project/cmake/installdata/common/common.txt

```diff
@@ -3,11 +3,14 @@ sounds/*
 system/keymaps/*
 system/library/*
 system/players/VideoPlayer/etc/*
+system/python/*
 system/shaders/*
 system/settings/*
 userdata/*
 system/addon-manifest.xml
 system/colors.xml
+system/IRSSmap.xml
 system/peripherals.xml
 system/playercorefactory.xml
+system/X10-Lola-IRSSmap.xml
```

Comment:
system/python shall always be empty on linux so why install it. replicating old bugs is not an acceptable answer.